pkg/apis/serving/k8s_validation.go

```diff
@@ -286,6 +295,50 @@ func validateEmptyDirFields(dir *corev1.EmptyDirVolumeSource) *apis.FieldError {
 	return errs
 }
 
+func validateHostPathVolumeSource(hostPath *corev1.HostPathVolumeSource) *apis.FieldError {
```

Comment:
Probably it would be good to add a comment here to show that this comes from the K8s code base.